### ISSUE TITLE
Unique assets in the Asset Browser can be duplicated only once

### ISSUE THREAD

**Issue by LB-ArturZieba** [o3de/o3de#13192]
**Describe the bug**
Assets in the asset browser can be duplicated only once, unless the *-copy#* file created via duplication is deleted. If multiple duplicates of a single asset are needed, the consecutive copies have to be individually duplicated.

**Steps to reproduce**
1. Right-click an asset (for example a custom .scriptcanvas asset) and select *Duplicate asset* from the dropdown menu.
2. Repeat step 1 on the same asset.

**Expected behavior**
Asset is duplicated.

**Actual behavior**
Nothing happens.

**Video**

https://user-images.githubusercontent.com/86952082/201924501-3909db46-a394-4b69-9c8d-9e4b3acc28d6.mp4

**Found in Branch**
Development (ba92844)

**Desktop**
 - Device: PC
 - OS: Windows
 - Version 11
 - CPU AMD Ryzen 5 3600
 - GPU NVIDIA GeForce RTX 2060 SUPER
 - Memory 16GB

**Comment by LB-ArturZieba** [comment]
Issue verified as fixed on the Development (cd0b0ec) build.
